### ISSUE TITLE
ItemsAdder items in recipes

### ISSUE THREAD

**Issue by fsosaaa** [PluginBugs/Issues-EpicBackpacks#31]
Well I think the title is clear
But it would be nice to be able to use IA items to craft the backpacks

**Comment by LoneDev6** [comment]
Good idea, but this will require a lot of work on the ItemsAdder code.
I hope to find the time in the next days.
Thanks a lot
